cg-spec/editors_draft.html

```diff
@@ -404,11 +407,23 @@ <h2>Triples and occurrences</h2>
         -->
       </pre>
 
-    </section>
+      <p>With this solution RDF-star takes three statements to annotate an occurrence. In contrast to RDF-star's approach RDF standard reification is defined as referring to occurrences. It is however underspecified insofar as it doesn't provide a way to denote the location - some graph, document or other source of a set of triples - in which a statement occurs, as the ":in" property in the example above does. RDF standard reification needs three statements to describe a triple and would need a forth statement to specify its location. Only then a fifth statement would be able to annotate such an occurrence. Note that the ":occurrenceOf" and ":in" properties in the example above serve only illustrative purposes as standardizing a proper vocabulary to denote occurrences is still an <a href="#rdf-star-vocabulary">open issue</a>. By comparison RDF/XML's <a href="https://www.w3.org/TR/rdf-syntax-grammar/#section-Syntax-reifying">ID</a> attribute implements reification without any deviation from RDF's abstract syntax and semantics and enables statement annotation without the need for any extra statements.</p>
+
+      <p>Referring to a specific occurrence, not to the triple as an abstract entity, is a common use case: provenance annotations on occurrences have until recently been the go-to example for RDF-star, as witnessed by the so-called <a href="#the-seminal-example">seminal example</a>. Attributed relations as in <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html#support-wikidata-wikibase-data-model">WikiData</a> or Property Graphs and also n-ary relations commonly refer to occurrences. It is important to be aware that when using RDF-star embedded triples such statement annotations have to provide references to both the occurrence and its containing set or graph or source as extra statements to be correct and complete, thereby putting a certain limit to the perceived simplicity and elegance of the RDF-star approach. Querying in SPARQL consequently also becomes more involved.</p>
```

Comment:
You should probably refrain from using "correct and complete" here. It has a very precise meaning for logicians and AI researchers, which has not much to do with the way you are using it here.

Reply:
Hard to assess for me as I don't understand how exactly a logician might take issue with this wording. Also this is not a normative section about the semantics. Would something like "accurate and comprehensive" be better?

Reply:
those terms are used to qualify a reasoning process, so they do not apply here. "accurate and comprehensive" sounds much better.